bedrock/security/templates/security/bug-bounty/faq-webapp.html

```diff
@@ -119,24 +122,105 @@ <h1 itemprop="name" class="title-shadow-box">Web Application Security Bug Bounty
             we do plan on adding to this list as we introduce our web security process to
             other sites not included in this list.</p>
 
-          <ul>
-            <li>bugzilla.mozilla.org</li>
-            <li>*.services.mozilla.com</li>
-            <li>aus*.mozilla.org</li>
-            <li>www.mozilla.com/org</li>
-            <li>www.firefox.com</li>
-            <li>www.getfirefox.com</li>
-            <li>addons.mozilla.org</li>
-            <li>services.addons.mozilla.org</li>
-            <li>versioncheck.addons.mozilla.org</li>
-            <li>download.mozilla.org</li>
-          </ul>
-
-          <p>Additionally, Mozilla services that handle reasonably sensitive user data
-            such as Sync and Hello are in scope for the bug bounty program.</p>
+          <pre>
+            Mozilla and Firefox websites
+            www.mozilla.com
+            www.mozilla.org
+            www.firefox.com
+            www.getfirefox.com
```

Comment:
You've used `pre` here as a shortcut for visual formatting but that isn't semantically appropriate for the content. These are lists with headings so they really should be marked up as such.